test/travis_script.sh

```diff
@@ -160,4 +162,19 @@ if [ -f .coverage ]; then
     coverage report
 fi
 
+echo '>>> Test 6. Reading manta file'
+
+
```

Comment:
Make sure to get a coverage report for the import using e.g.
`sed 's/import_4/mate_1/'`
Also make sure the filename (`.coverage_mate_1` in the example above) is added to the `coverage combine` line.

Also: Test 6 should come before Finalizing.